pkgs/development/libraries/gstreamer/devtools/default.nix

```diff
@@ -54,6 +63,7 @@ stdenv.mkDerivation (finalAttrs: {
       stripLen = 2;
```

Comment:
How would this patch work with the updated `src`? We strip the prefix from the paths.

Reply:
given that it worked I assumed the builder would enter the `sourceRoot` before applying the patches, though admittedly I didn't check